ci-operator/step-registry/openstack/provision/bastionproxy/openstack-provision-bastionproxy-commands.sh

```diff
@@ -68,6 +68,7 @@ bastion_fip="$(openstack floating ip create -f value -c floating_ip_address \
 >&2 echo "Created floating IP ${bastion_fip}"
 >&2 openstack server add floating ip "$server_id" "$bastion_fip"
 echo ${bastion_fip} |awk '{print $2}' >> ${SHARED_DIR}/DELETE_FIPS
+echo ${bastion_fip} |awk '{print $2}' > ${SHARED_DIR}/BASTION_FIP
```

Comment:
```suggestion
echo ${bastion_fip} > ${SHARED_DIR}/BASTION_FIP
```
Same with the above line. This causes us to leak FIPs.